bootstrap-scripts/raspberry-pi-setup.sh

```diff
@@ -62,4 +62,6 @@ echo Install node bindings for cmake
 sudo npm install -g cmake-js
 
 # Configure abigail-device service
-sudo cp $mydir/config-files/abigail-device.service /lib/systemd/system
+# Note: the ugly bash expansion below replaces all `/` with `\/` so that sed is
+# happy.
+sudo sed "s/<SOURCEDIR>/${source_dir////\\/}/g" $source_dir/config-files/abigail-device.service.in > /lib/systemd/system/abigail-device.service
```

Comment:
I like having a templated approach. Can you explain how this 
```
s/<SOURCEDIR>/${source_dir////\\/}/g
```
works? (especially '////')